roles/docker_health_monitor/tasks/main.yml

```diff
@@ -0,0 +1,21 @@
+---
+  - name: create script directory
+    file:
+      path: "{{ script_directory }}"
+      state: directory
+      owner: "{{ api_user }}"
+      group: "{{ api_group }}"
+      mode: '0775'
+
+  - name: copy script
+    copy:
+      src: "{{ script_name }}"
+      dest: "{{ script_path }}"
```

Comment:
Dest can be just `{{ script_directory }}`. In addition:

> If dest is a non-existent path and if either dest ends with "/" or src is a directory, dest is created.

So the previous task can be removed as well.

Ref: https://docs.ansible.com/ansible/latest/modules/copy_module.html